package.json

```diff
@@ -17,13 +21,17 @@
     "tsdoc": "bin/index.js"
   },
   "scripts": {
+    "build": "node rmdir.mjs && node esbuild.mjs && npm run ts-declaration",
+    "contributors:add": "all-contributors add",
+    "contributors:check": "all-contributors check",
+    "contributors:generate": "all-contributors generate",
+    "docs": "node rmdir.mjs && node esbuild.mjs && node bin/index.js --src=src/lib/*  --repo=https://github.com/peterpeterparker/tsdoc-markdown && prettier --write ./README.md",
     "format": "prettier --write .",
     "lint": "eslint --max-warnings 0 .",
-    "build": "node rmdir.mjs && node esbuild.mjs && npm run ts-declaration",
-    "ts-declaration": "tsc --emitDeclarationOnly --outDir dist/types",
-    "test": "jest",
+    "prepublish": "npm run format && npm run lint && npm run test && npm run build",
```

Comment:
Do we need this additional `prepublish` script?

Can you also preserve the order of the scripts? it will be easier to compare what's changed.

Reply:
prepublish was just for ease of mind, I'll remove it